benefit/VERSION.yaml

```diff
@@ -1 +1,2 @@
-TAG: 5.15.9
\ No newline at end of file
+TAG: 5.15.10
+
```

Comment:
Remove whitespace. I'm about to deploy my ticket to 5.15.10 so might be good to bump this up again